packages/test/test-end-to-end-tests/src/test/gc/gcSweepUnreferencePhases.spec.ts

```diff
@@ -84,39 +90,32 @@ describeCompat("GC sweep unreference phases", "NoCompat", (getTestObjectProvider
 		settings["Fluid.GarbageCollection.TestOverride.SweepTimeoutMs"] = sweepTimeoutMs;
 	});
 
-	it("GC nodes go from referenced to unreferenced to inactive to sweep ready to swept", async () => {
+	it("Unreferenced objects follow the sequence [unreferenced, tombstoned, deleted]", async () => {
 		const mainContainer = await provider.makeTestContainer(testContainerConfig);
 		const mainDataStore = (await mainContainer.getEntryPoint()) as ITestDataObject;
 		await waitForContainerConnection(mainContainer);
 
 		const { container, summarizer } = await loadSummarizer(mainContainer);
 
-		// create datastore and blob
```

Comment:
@agarwal-navin want me to add blobs back in, or maybe run the whole test twice, once with DataStore once with Blob? I might do it as a follow-up, I've got a list.

Reply:
Out of curiosity, why did you have to remove blobs?

Reply:
I didn't. It made it easier for me to read through / update the tests.  Less fiddling around while getting them to pass.  I'll add it back as a follow-up, not in this PR.